### ISSUE TITLE
Missed last line

### ISSUE THREAD

**Issue by ai** [hhsnopek/code-frame#12]
``` js
framer('\nfunction (foo) {\n\tconsole.log(foo\n}', 3, 24)
//   2 | function (foo) {
// > 3 |         console.log(foo
//     |                        ^
//   4 |
```

Why line 4 is empty? Do I understand correctly, that it is a issue and it should be with `}` — `4 | }`?

**Comment by ai** [comment]
Here is a code, seems like code was forgotten =^_^= https://github.com/hhsnopek/code-frame/blob/master/lib/index.js#L31

But anyway we need to rewrite this code if we decide to fix #13.

**Comment by hhsnopek** [comment]
Closing this as it was an intention, discussion to continue in #13
